src/PageBuilder.php

```diff
@@ -129,28 +133,28 @@ protected function loadCollection($dir)
      *
      * @return Page
      */
-    protected function loadCachePage($file)
+    protected function loadCachePage($file): Page
     {
         $page = null;
         $cache = $this->configCacheFactory->cache(
             $this->getCacheFile($file),
-            function (ConfigCacheInterface $cache) use ($file, &$page) {
+            function (ConfigCacheInterface $cache) use ($file, &$page): void {
                 $page = $this->loadPage($file);
-                $str = serialize($page);
+                $str = \serialize($page);
                 $cache->write($str, [new FileResource($this->root . $file)]);
             }
         );
 
         if (!$page) {
-            $page = unserialize(file_get_contents($cache->getPath()));
+            $page = \unserialize(\file_get_contents($cache->getPath()));
```

Comment:
Want to change these `serialize` and `unserialize` calls to use `Bolt\Common\Serialization`?

Reply:
Derp … totally missed these :disappointed: